modules/Clinical/domains/demographics.yaml

```diff
@@ -57,27 +45,43 @@ enums:
       "Unknown":
         description: Not known, not observed, not recorded, or refused
 
+  SexEnum:
+    title: sex
+    permissible_values:
+      "Female":
+        description: Female
+      "Male":
+        description: Male
+      "Not Reported":
+        description: Not provided or available
+      "Unknown":
+        description: Not known, not observed, not recorded, or refused
+
 classes:
   Demographics:
     description: Information about the demographics
     attributes:
       ETHNIC_GROUP:
         range: EthnicGroupEnum
         required: true
+        title: "Ethnic Group"
         description: Ethnic group of the participant (caDSR:2192201) (Aligns to CDRC Standard CDE)
         slot_uri: caDSR:2192201
       GENDER_IDENTITY:
         range: GenderIdentityEnum
         required: true
+        title: "Gender Identity"
         description: Gender identity of the participant (caDSR:2192202) (Aligns to CDRC Standard CDE)
         slot_uri: caDSR:2192202
```

Comment:
I think this term is deprecated in. CADSR... we might weant. to expand our descriptions in case the cadsr term becomes unavaliable